docs/building-apps/canister-management/settings.mdx

```diff
@@ -13,7 +13,7 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
 Each <GlossaryTooltip>canister</GlossaryTooltip> has a group of possible settings that control its behavior. Only a controller of the canister can read and modify a canister's settings.
 
-Currently, there are eight canister setting fields:
```

Comment:
This PR should be merged when the roll-out is complete and changelog should be updated before merging.